client/components/tinymce/plugins/contact-form/style.scss

```diff
@@ -78,3 +78,20 @@
 		color: $orange-fire;
 	}
 }
+
+.editor-contact-form-modal-field__edit-wrapper {
+	height: 100%;
+
+	button {
```

Comment:
Do you mind adding another class to the button instead. We can then do something like:
```
.editor-contact-form-modal-field__edit-wrapper {
    height: 100%;
}
.editor-contact-form-modal-field__edit-wrapper-button {
   width: 100%;
}
```